src/System.Runtime.Extensions/tests/System/Environment.Exit.cs

```diff
@@ -42,6 +42,15 @@ public static void CheckExitCode(int expectedExitCode)
 
         [Theory]
         [MemberData(nameof(ExitCodeValues))]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.UapNotUapAot, "RemoteExecutor behaves different in UAP.")]
```

Comment:
The if check will avoid a NullReferenceException in UAP because RemoteExecutorTestBase in UAP returns a RemoteInvokeHandle with a `null` process: https://github.com/dotnet/corefx/blob/master/src/CoreFx.Private.TestUtilities/src/System/Diagnostics/RemoteExecutorTestBase.uap.cs#L74

Therefore we will need to avoid using the returned process in UAP as it will be `null`. So when doing [Assert.True(p.WaitForExit(30 * 1000)](https://github.com/dotnet/corefx/blob/master/src/System.Runtime.Extensions/tests/System/Environment.Exit.cs#L31) will cause a `NullReferenceException` because `p` is `null` in UAP.